.gitmodules

```diff
@@ -10,20 +10,12 @@
 #     rm -rf path_to_submodule
 
 
-[submodule "submodules/HaRe"]
-	path = submodules/HaRe
-	# url = https://github.com/bubba/HaRe.git
-	url = https://github.com/alanz/HaRe.git
-
 [submodule "submodules/cabal-helper"]
 	path = submodules/cabal-helper
-	# url = https://github.com/arbor/cabal-helper.git
-	url = https://github.com/alanz/cabal-helper.git
 	# url = https://github.com/DanielG/cabal-helper.git
+	# Change this back once https://github.com/DanielG/cabal-helper/pull/85/ merged
+	url = https://github.com/bubba/cabal-helper.git
 
 [submodule "submodules/ghc-mod"]
 	path = submodules/ghc-mod
-	# url = https://github.com/arbor/ghc-mod.git
-	# url = https://github.com/bubba/ghc-mod.git
-	url = https://github.com/alanz/ghc-mod.git
-
+	url = https://github.com/fendor/ghc-mod.git
```

Comment:
Need an issue which details what to do to remove this?

Reply:
Good idea, we want to subsume `ghc-project-types` to make us independent of ghc-mod.

Reply:
Moreover, if we take `ghc-project-types` and turn it into its own project, we are almost ready to go to hackage.

Reply:
I did discuss that with @DanielG , he is happy for us to move it elsewhere, possibly as a sub-project inside haskell-ide-engine.